brainstormforce/ultimate-addons-for-gutenberg
# Default Spacing - Removed default spacing added for parent & child for price-list & icon list.

### Description

- Default spacing between child and parent in Price List - https://d.pr/v/gZDACV
- Spacing issue in the editor for Icon List - https://d.pr/v/vUa5s4

### Screenshots

- https://share.getcloudapp.com/rRu1KzdE

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop Price List and Icon List in editor.
- Check is it renders the same as frontend in editor.

### Checklist:
